### PR TITLE
feat: start apiserver in standalone mode

### LINKED ISSUE #606: Start APIServer in `start-all.sh` script
We need  to start APIServer in standalone mode

### PR DESCRIPTION
**Pull request by dl239**

### CODE REVIEW

**Comment by vagetablechicken** [comment]
ut got 'Segmentation fault'

**Review by vagetablechicken — APPROVED** [review]

File: src/sdk/sql_cluster_router.h [hunk 1]
```diff
@@ -190,6 +190,17 @@ class SQLClusterRouter : public SQLRouter {
     std::shared_ptr<hybridse::sdk::Schema> GetTableSchema(
         const std::string& db, const std::string& table_name) override;
 
+    base::Status HandleSQLCreateProcedure(hybridse::node::CreateProcedurePlanNode* plan,
+            const std::string& db, const std::string& sql,
+            std::shared_ptr<::openmldb::client::NsClient> ns_ptr);
+
+    base::Status HandleSQLCreateTable(hybridse::node::CreatePlanNode* create_node, const std::string& db,
```
Comment (aceforeverd) [inline src/sdk/sql_cluster_router.h:197]: By convention, it should write as `HandleSqlCreateTable`

Comment (aceforeverd) [reply src/sdk/sql_cluster_router.h:197]: maybe resolve later

**Review by aceforeverd — APPROVED** [review]

File: src/sdk/sql_cluster_router.h (hunk 1 above)
Comment (dl239) [reply src/sdk/sql_cluster_router.h:197]: yep. `SQLXXX` name-style is exists in many places. we will update these in another pr

**Pull request merged**